tests/testthat/test_surv_measures.R

```diff
@@ -0,0 +1,43 @@
+context("survival measures")
+
```

Comment:
This is just personal preference (in order to let the base group of tests run quickly), we could also join the two files.